submissions/adekemiAwomolo/html-forms/product.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
```

Comment:
Kindly work on your existing product page you have created in your `html` folder

Reply:
@adaugochi Good afternoon ma, i am so sorry about my commit messages ma, i have worked on all the corrections you asked ma.Kindly review ma, thank you very much ma.